lib/internal/modules/esm/resolve.js

```diff
@@ -714,8 +715,8 @@ function resolveAsCommonJS(specifier, parentURL) {
 
 function defaultResolve(specifier, context = {}, defaultResolveUnused) {
   let { parentURL, conditions } = context;
-  if (manifest) {
-    const redirects = manifest.getRedirector(parentURL);
+  if (parentURL && policy && policy.manifest) {
```

Comment:
Likewise here... I know we haven't started adopting the `?.` syntax throughout core yet, but would be good to know if we can...

Reply:
fixed